app/controllers/subnets_controller.rb

```diff
@@ -58,7 +58,7 @@ def freeip
 
   def import
     proxy = SmartProxy.find(params[:smart_proxy_id])
-    @subnets = Subnet.import(proxy)
+    @subnets = Subnet::Ipv4.import(proxy)
```

Comment:
We might want to show something on the UI specifying we're importing only IPv4 subnets (or is IPv6 import coming later)?